ts/features/itwallet/identification/screens/ItwIdentificationNfcInstructionsScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  ContentWrapper,
+  Divider,
+  ListItemHeader,
+  ListItemInfo
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { Linking } from "react-native";
+import { RNavScreenWithLargeHeader } from "../../../../components/ui/RNavScreenWithLargeHeader";
+import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { FooterStackButton } from "../../common/components/FooterStackButton";
+
+export const ItwIdentificationNfcInstructionsScreen = () => {
+  const navigation = useIONavigation();
+
+  const handleOpenSettingsPress = async () => {
+    await Linking.openSettings();
+  };
+
+  const handleClosePress = () => {
+    navigation.pop();
+  };
+
+  return (
+    <RNavScreenWithLargeHeader
+      title={{ label: I18n.t("features.itWallet.identification.nfc.title") }}
+      description={I18n.t("features.itWallet.identification.nfc.description")}
+      fixedBottomSlot={
+        <FooterStackButton
+          primaryActionProps={{
+            label: I18n.t("features.itWallet.identification.nfc.primaryAction"),
+            onPress: handleOpenSettingsPress
+          }}
+          secondaryActionProps={{
+            label: I18n.t(
+              "features.itWallet.identification.nfc.secondaryAction"
+            ),
+            onPress: handleClosePress
+          }}
+        />
+      }
+    >
+      <ContentWrapper>
+        <ListItemHeader
+          label={I18n.t("features.itWallet.identification.nfc.header")}
+        />
+        <ListItemInfo
+          label={"Step 1"}
```

Comment:
What do you think to add a locale also there? 
for example it could be features.itWallet.identification.nfc.steps.1.{title,value}

Reply:
Since we use the english word also for the italian version, I supposed that the locale was not necessary. 
But you are right, it can be useful in case we decide to add translations later.